### PR TITLE
Stripping newline character from assesment

### PR DESCRIPTION
**Pull request by alisha-nas**
Removed the newline character for better formatting

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,8 +17,16 @@
   </h2>
 </div>
 
+<<<<<<< HEAD
```
Comment (cloudyyoung) [inline README.md:20]: This seems to be an unresolved conflict

Comment (skilledd007) [reply README.md:20]: Yep.

**Review by skilledd007 — CHANGES_REQUESTED** [review]

**Review by tim-macphail — CHANGES_REQUESTED** [review]
Looks like a bunch of extra files were created somehow. All we really care about is the changes in file_handler.py so you can go ahead and delete all the files with 2 in the name, add, commit, push again and it will update this pull request (make sure you're on this branch)

File: .gitattributes 2 [hunk 2]
```diff
@@ -0,0 +1 @@
+*.pdf filter=lfs diff=lfs merge=lfs -text
```
Comment (tim-macphail) [inline .gitattributes 2:1]: duplicate file can be deleted

**Review by tim-macphail — APPROVED** [review]

**Review by tim-macphail — APPROVED** [review]

**Review by cloudyyoung — APPROVED** [review]

**Pull request merged**